Help/FAQ.rst

```diff
@@ -55,6 +55,15 @@ There are also generic bears, which can be applied language independently on
 your code. Their capabilities and information can be seen
 `here <https://coala.io/#/languages>`__.
 
+What is a bear anyway?
+----------------------
+
+A bear is a set of rules that is used by coala packages to check for
```

Comment:
I think plural would be better since it a bear can have multiple lint checks in place or just one so change this to `check` -> `check(s)`